### ISSUE TITLE
Default step size for sliders too small?

### ISSUE THREAD

**Issue by emily-phet** [phetsims/ohms-law#164]
Was working with Ohm's Law to make recordings for the description course. Noticed that it takes like 30 keypresses to traverse the whole slider. I don't think that's what we originally designed. When this sim gets worked on again, would be good to check that.

**Comment by terracoda** [comment]
The default number of step sizes should be around 18.
The first step goes from 0.1 Volts to 0.5 Volts, and then up by 0.5 Volts with each step up to 9.0 Volts.

Seems to be working on master and in the published version.

**Comment by terracoda** [comment]
Oh but for Resistance the range is much wider 1000 ohms.
The first step goes from 10 to 10 Ohms and then up by 20 Ohms with each step up to 1000 Ohms.

That indeed results in 50 key presses at the default step size. 
The Shift modifier goes to 1 Ohm which would be 999 key presses.
With page up and page down shortcuts the step size is 99 Ohms - I don't think we we can adjust larger steps. I think it is just the max spread divided by 10.

Rounding to the nearest 20 Ohms happens if you go from using the page up and page down keys back to using the default key press, so I think it is implemented as intended.

**Comment by arouinfar** [comment]
> so I think it is implemented as intended.

We're getting ready to start work on the 1.5 release, and I'm taking a look through open issues. The behavior on main is still as @terracoda describes, so I'll close this issue.
